packages/lib/src/cql-translator-service/ast-to-cql-translator.ts

```diff
@@ -13,23 +13,17 @@ import {
     cqltemplate,
     criterionMap,
 } from "./cqlquery-mappings";
-import { getCriteria, resolveAstSubCatagories } from "../stores/catalogue";
+import { resolveAstSubCatagories } from "../stores/catalogue";
 import type { MeasureItem } from "../types/backend";
 
-let codesystems: string[] = [];
```

Comment:
The `codesystems` variable is still used elsewhere, no?

Reply:
Yes it is in line 504, 505 and so on. It is a global variable and not reset after the function is called. However, having CodeSystem which are not used is not crashing anything. Your are right, the code style there is bad. However, we should improve it in the future.

Reply:
Ah my bad, I misread the diff. Looks good.